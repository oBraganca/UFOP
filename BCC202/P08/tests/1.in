1
9
ana 25
joao 30
carlos 20
maria 25
pedro 30
jose 35
joana 35
maura 27
ana 29
