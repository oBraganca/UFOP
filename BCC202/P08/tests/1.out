Teste 1:
ana 25
ana 29 
carlos 20 
joana 35
joao 30 
jose 35 
maria 25 
maura 27 
pedro 30